arm/Microsoft.ServiceFabric/clusters/applications/deploy.bicep

```diff
@@ -0,0 +1,94 @@
+  rollingUpgradeMonitoringPolicy: {
+    failureAction: !empty(properties.upgradePolicy.rollingUpgradeMonitoringPolicy.failureAction) ? properties.upgradePolicy.rollingUpgradeMonitoringPolicy.failureAction : 'Manual'
+    healthCheckRetryTimeout: !empty(properties.upgradePolicy.rollingUpgradeMonitoringPolicy.healthCheckRetryTimeout) ? properties.upgradePolicy.rollingUpgradeMonitoringPolicy.healthCheckRetryTimeout : null
+    healthCheckStableDuration: !empty(properties.upgradePolicy.rollingUpgradeMonitoringPolicy.healthCheckStableDuration) ? properties.upgradePolicy.rollingUpgradeMonitoringPolicy.healthCheckStableDuration : null
+    healthCheckWaitDuration: !empty(properties.upgradePolicy.rollingUpgradeMonitoringPolicy.healthCheckWaitDuration) ? properties.upgradePolicy.rollingUpgradeMonitoringPolicy.healthCheckWaitDuration : null
+    upgradeDomainTimeout: !empty(properties.upgradePolicy.rollingUpgradeMonitoringPolicy.upgradeDomainTimeout) ? properties.upgradePolicy.rollingUpgradeMonitoringPolicy.upgradeDomainTimeout : null
+    upgradeTimeout: !empty(properties.upgradePolicy.rollingUpgradeMonitoringPolicy.upgradeTimeout) ? properties.upgradePolicy.rollingUpgradeMonitoringPolicy.upgradeTimeout : null
+  }
+  upgradeMode: !empty(properties.upgradePolicy.upgradeMode) ? properties.upgradePolicy.upgradeMode : 'Invalid'
+  upgradeReplicaSetCheckTimeout: !empty(properties.upgradePolicy.upgradeReplicaSetCheckTimeout) ? properties.upgradePolicy.upgradeReplicaSetCheckTimeout : null
+}
+
+module pid_cuaId '.bicep/nested_cuaId.bicep' = if (!empty(cuaId)) {
+  name: 'pid-${cuaId}'
+  params: {}
+}
+
+resource serviceFabricCluster 'Microsoft.ServiceFabric/clusters@2021-06-01' existing = {
+  name: serviceFabricClusterName
+}
+
+resource applications 'Microsoft.ServiceFabric/clusters/applications@2021-06-01' = {
+  name: applicationName
+  location: location
+  parent: serviceFabricCluster
+  tags: tags
+  identity: !empty(identity) ? identity_var : null
+  properties: {
+    managedIdentities: !empty(properties.managedIdentities) ? propertiesManagedIdentities_var : null
+    maximumNodes: contains(properties, 'maximumNodes') ? properties.maximumNodes : 0
+    metrics: !empty(properties.metrics) ? propertiesMetrics_var : null
+    minimumNodes: contains(properties, 'minimumNodes') ? properties.minimumNodes : 0
+    parameters: !empty(properties.parameters) ? properties.parameters : null
+    removeApplicationCapacity: contains(properties, 'removeApplicationCapacity') ? properties.removeApplicationCapacity : false
+    typeName: !empty(properties.typeName) ? properties.typeName : null
+    typeVersion: !empty(properties.typeVersion) ? properties.typeVersion : null
+    upgradePolicy: !empty(properties.upgradePolicy) ? upgradePolicy_var : null
+  }
+}
+
+// Output
+@description('The resource Id of the application.')
+output applicationResourceId string = applications.id
```

Comment:
Add the default outputs name & resourceGroup with descriptions

Reply:
Added.